# SER_variation_in_SARS-CoV-2
COVID-19 is a global pandemic, and its influential variants continue to emerge. Effective information for assisting to identify and even predict new mutations is helpful and urgent. Changes in viral synonymous sites are typically regarded as a concomitant mutation and their roles on providing evolutionary insights were mostly ignored and not fully explored. Here, synonymous evolutionary rate (SER) was estimated and based on it, the relationship between the viral RNA and host protein was inferred, as well as the pattern of characteristic mutations found in different lineages were investigated. The findings show that SER varies across the genome and the main contributor of the variation is codon related factors. Conserved motifs identified from the angle of SER are found to be related to host RNA transport and regulation. Importantly, the majority of the existing fixed characteristic mutations for five important lineages (Alpha, Beta, Gamma, Delta and Omicron) are significantly enriched in partially constrained regions. Those results give unique information for the evolutionary and functional dynamics of the SARS-CoV-2 virus from the perspective of synonymous mutations and provide potential useful information for timely and precise control of SARS-CoV-2 epidemic.
